README.md

```diff
@@ -0,0 +1,47 @@
+# NJOY metaconfigure
+The metaconfigure package contains a suite of Python scripts to assist in the generation of build system configuration files for NJOY projects.
+
+## Updating metaconfigure in a project
+**Note:** that all the subprojects/dependencies must be updated prior to updating the super project.
+
+These commands (unless otherwise specified) should be run from the top level of a project.
+1. Create a branch
+2. Download ("fetch") subprojects
+   ```bash
+   python metaconfigure/fetch_subprojects.py
+   ```
+3. Update the metaconfigure subtree
+   ```bash
+   git subtree pull --squash --prefix metaconfigure https://github.com/njoy/metaconfigure.git master
+   ```
+4. Delete old `description.json` file.
+   ```bash
+   rm metaconfigure/description.json
+   ```
+5. Generate new `metaconfigure/description.json` file. This is done in Python.
+   ```python
+   import metaconfigure
+   args = {"name": "project-name", "language": "c++", "is external project": False, "include path": "src/include"}
```

Comment:
yeah, `metaconfigure`, in particular description generation, needs a manual. That's going to require a considerable investment of time. After this years deliverables are complete, I plan to return to documentation in the form of a LA-UR